### PR TITLE
Configure sensitivity

### PR DESCRIPTION
**Pull request by just-boris**
In my application, I have a requirement to make slides more responsible. I think it would be better for everybody to have a property to customize hysteresis behavior. Current behavior is preserved as defaultProp value.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +240,7 @@ export default MyComponent;
 | children | node | | all | Use this property to provide your slides. |
 | containerStyle | object | `{}` | all | Whether or not the auto complete is animated as it is toggled. |
 | disabled | bool | `false` | all | If `true`, it will disable touch events. This is useful when you want to prohibit the user from changing slides. |
+| hysteresis | float | `0.6` | all | Configure hysteresis between slides. This value determines how far should user swipe to switch slide. |
```
Comment (oliviertassinari) [inline README.md]: That's only exposed on the browser platform. Use *browser* instead of *all*.

File: src/SwipeableViews.spec.js [hunk 2]
```diff
@@ -29,6 +29,77 @@ describe('SwipeableViews', () => {
     });
   });
 
+  describe('prop: hysteresis', () => {
```
Comment (oliviertassinari) [inline src/SwipeableViews.spec.js]: I should start adding some e2e tests infrastructure. That would make a great test.

**Comment by just-boris** [comment]
Done.

**Comment by oliviertassinari** [comment]
@just-boris Thanks!

**Comment by just-boris** [comment]
@oliviertassinari could you make a new realease, please?
Currently, I depend on my fork to have this feature.

**Comment by oliviertassinari** [comment]
@just-boris Yes, I'm on it.

**Pull request merged**